### PR TITLE
add precision option

### PR DESCRIPTION
**Pull request by Neodelf**
Add precision option to #to_words

### CODE REVIEW

**Comment by dblock** [comment]
How do I exercise this from code?

**Comment by Neodelf** [comment]
@dblock, you might execute `7.1.to_words(precision: 3)`

**Comment by dblock** [comment]
I would add a test that calls that @Neodelf. Next this needs a README and a CHANGELOG update, please. And the build to pass.

**Comment by dblock** [comment]
I would just remove rbx from travis.
If a newer version of JRuby doesn't just work, remove that too.

**Comment by Neodelf** [comment]
@dblock you want to remove 'rbx' and 'jruby' versions in '.travis.yml`?
Tests covers new added code (but only russian version or is not true?). Yes, I will change README and CHANGELOG in very soon.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-## 0.11.6 (Next)
+## 0.10.6 (Next)
 
-  * Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md]: Put this line back ;)

**Comment by dblock** [comment]
I just want the build to pass, and I don't care enough about JRuby and definitely not about RBX to waste your time. From tests it's not clear that you can do `to_words(precision: ...)`, so I was hoping for one just like that in any language.

**Comment by Neodelf** [comment]
> From tests it's not clear that you can do to_words(precision: ...)
I just followed convention about existed tests to order to new tests are easy to fit in existence

File: CHANGELOG.md (hunk 1 above)
Comment (Neodelf) [inline CHANGELOG.md]: ok

**Comment by dblock** [comment]
Looking at existing tests maybe I'm asking for something we don't need. I'm happy with the PR as long as you can please help fix the build?

**Comment by dblock** [comment]
Just remove rbx or add it to `allow_failures`, it's not worth it.

**Comment by Neodelf** [comment]
@dblock yes, I removed it)

File: spec/numbers_and_words/float/fixture_examples/ru.yml [hunk 2]
```diff
@@ -15,3 +15,16 @@ to_words:
     21.77: двадцать одна целая и семьдесят семь сотых
     111.999: сто одиннадцать целых и девятьсот девяносто девять тысячных
     4242.7463: четыре тысячи двести сорок двe целых и семь тысяч четыреста шестьдесят три десяти тысячных
+  fractions_with_precision:
+    options:
+      :precision: 2
```
Comment (jlduran) [inline spec/numbers_and_words/float/fixture_examples/ru.yml]: Shouldn't this be `precision: 2`? (as well as the others)

Comment (Neodelf) [inline spec/numbers_and_words/float/fixture_examples/ru.yml]: oh, yes, it's right! I changing it

**Comment by jlduran** [comment]
I made a couple minor suggestions, but it's OK as it is.

**Comment by dblock** [comment]
Merged.

**Pull request merged**